tools/README.md

```diff
@@ -362,4 +362,22 @@ For example, in order to translate a windows-related Sigma rule, one would use:
 
 ```bash
 tools/sigmac -t devo -c tools/config/devo-windows.yml rules/windows/sysmon/sysmon_wmi_susp_scripting.yml
-```
\ No newline at end of file
+```
+
+### Datadog
+The Datadog backend currently supports converting Sigma files to the [Log Search Syntax](https://docs.datadoghq.com/logs/explorer/search_syntax/)
+with the identifier `datadog-logs`. This query can be used in the Security Monitoring product.
+
+#### Config file
+The Datadog backend does not require a config file.
+If you choose to add one, you can specify facets in addition to the existing features.
+While attributes will be queried with `@my-attribute:attribute_value` specified facets will be queried with `my-facet:service_value`.
```

Comment:
Do we need to require the users to do that? (see below, it's probably better if we provide OOTB facets as good defaults, and offer the possibility to tweak it in case our Sigma supports lags behind new OOTB facets that can be added in the future)

Reply:
In fact, it's a little more complicated than that.

For attributes, you always need `@`
For tags, you never need [at]

Special cases:
 - `host` is an attribute but does not require [at]
 - `service` is both an attribute and a tag, it does not require [at]
 - ~~`source` is a tag, does not require [at], so it's not a special case after all~~

Reply:
I think we were confused about the tags vs facets naming. We made the naming consistent with Datadog's doc in a21fe1e and now only mention tags in our code. We added all the tags mentioned in https://docs.datadoghq.com/getting_started/tagging/#introduction but we can probably add more. I could not find an exhaustive list of OOTB tags though.